contracts/JBETHPaymentTerminalStore.sol

```diff
@@ -419,9 +415,9 @@ contract JBETHPaymentTerminalStore {
     }
 
     // Store the new amount.
-    usedDistributionLimitOf[_projectId][fundingCycle.number] = _newUsedDistributionLimitOf;
+    usedDistributionLimitOf[_projectId][fundingCycle.configuration] = _newUsedDistributionLimitOf;
```

Comment:
I think here we want .number

Reply:
the distributionLimit is a per-funding cycle basis, not a per-configuration basis. Projects should get the limit again as funding cycles roll over without explicit reconfiguration.

overflowAllowance, on the other hand, i think might be better as per configuration. i.e. "the DAO can access 30 ETH from overflow" and it doesnt reset each funding cycle.

Reply:
Oh yeah now that I think back this was a cause of confusion for me. Definitely makes sense to look up distributionLimit on a per-funding cycle basis.

Reply:
Made the change from funding cycle configuration to number here but the controller and terminal that @drgorillamd is handling will need some updates. 

controller.remainingDistributionLimitOf method
terminal.distributionLimitOf mapping
terminal._configure method

will point those out in the respective PRs